doc/how-to/getting_started_db.rst

```diff
@@ -1,13 +1,282 @@
-.. _getting_started_db:
+..  _getting_started_db:
 
-================================================================================
 Creating your first Tarantool database
-================================================================================
+======================================
 
-First, let's install Tarantool, start it, and create a simple database.
+In this tutorial, you connect to Tarantool instances using the :ref:`tt CLI <tt-installation>` utility,
+create a database, and establish a remote connection using the :ref:`net.box <net_box-module>` module.
 
-You can install Tarantool and work with it locally or in Docker.
+Installing Tarantool
+--------------------
 
-.. include:: using_docker.rst
+For production purposes, we recommend that you install Tarantool via the
+`official package manager <http://tarantool.org/download.html>`_.
+You can choose one of three versions: LTS, stable, or beta.
+An automatic build system creates, tests, and publishes packages for every
+push into a corresponding branch at
+`Tarantool's GitHub repository <https://github.com/tarantool/tarantool>`_.
```

Comment:
This is not exactly so. We don't offer per-push builds via our repositories anymore.

Anyway, this is a technical detail and I would just remove this sentence instead of describing our deployment process. It looks irrelevant for this guide.